.danger/__fixtures__/changeset/tiny-robot-hands.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/avatar': major
+'@twilio-paste/core': major
+---
+
+Test changeset with major core
```

Comment:
Question: did the test work? you changed the upgrade guide so i cant tell.

Reply:
Danger only looks at the real changesets directory, so it would never fail based on this fixture being added. I am relying on the jest tests right now to prove it works.